dvc/remote/local/__init__.py

```diff
@@ -56,11 +57,19 @@ class RemoteLOCAL(RemoteBASE):
         "reflink": System.reflink,
     }
 
+    SHARED_MODE_MAP = {None: (0o644, 0o755), "group": (0o664, 0o775)}
```

Comment:
These look the same.

Reply:
But they are not. What is your point? Using macros?

Reply:
Discussed in PMs.